internal/provider/resource_saml_idp.go

```diff
@@ -61,7 +66,7 @@ func resourceWizSAMLIdP() *schema.Resource {
 			"domains": {
 				Type:        schema.TypeList,
 				Required:    true,
-				Description: "A list of domains the IdP handles.",
+				Description: "A list of domains the IdP handles. This block needs to be set to accomodate for update lifecycle operations. If not used then specify an empty list (`[]`)",
```

Comment:
It looks like domain can be made optional.  In prior runs, I recall the field being required, but now it appears to be optional.

Reply:
See https://github.com/AxtonGrams/terraform-provider-wiz/pull/42#discussion_r1036255152